engine/src/flutter/lib/ui/window.dart

```diff
@@ -164,10 +164,7 @@ class FlutterView {
   /// See also:
   ///
   ///  * [physicalSize], which returns the current size of the view.
-  // TODO(goderbauer): Wire this up so embedders can configure it. This will
-  //   also require to message the size provided to the render call back to the
-  //   embedder.
-  ViewConstraints get physicalConstraints => ViewConstraints.tight(physicalSize);
+  ViewConstraints get physicalConstraints => _viewConfiguration.viewConstraints;
```

Comment:
Since the embedders don't actually have this implemented yet, I think we should fallback if the constraints are empty.

```suggestion
  ViewConstraints get physicalConstraints => _viewConfiguration.viewConstraints.isEmpty
      ? ViewConstraints.tight(physicalSize)
      : _viewConfiguration.viewConstraints;
```

I would add an `isEmpty` getter to `ViewConstraints`:
```dart
bool get isEmpty => minWidth == 0.0 && maxWidth == 0.0 && minHeight == 0.0 && maxHeight == 0.0;
```

Reply:
If possible, I'd prefer that we don't send incorrect metrics. I'd prefer we update  embedders to send metrics where the constraints are just the size converted to tight constraints. Effectively that pushes down the current constraints logic to embedders.

Reply:
Correct me if I'm wrong, but people could have custom embedders, right? We wouldn't want this to break them

Reply:
Opening this back up.  Even after trying to get the tests passing just getting the embedders to pass in proper view constraints I had to fall back to what @vashworth suggested to get the tests to pass.

Reply:
Updated tester_main.cc and tests seem to be passing now.